src/System.Security.Cryptography.X509Certificates/tests/CertTests.cs

```diff
@@ -363,6 +363,43 @@ public static void ExportPublicKeyAsPkcs12()
             }
         }
 
+        [Fact]
+        public static void X509Certificate2WithT61String()
+        {
+            string certSubject = @"E=mabaul@microsoft.com, OU=Engineering, O=Xamarin, S=Massachusetts, C=US, CN=test-server.local";
+
+            using (var cert = new X509Certificate2(TestData.T61StringCertificate))
+            {
+                Assert.Equal(certSubject, cert.Subject);
+                Assert.Equal(certSubject, cert.Issuer);
+
+                int snlen = cert.GetSerialNumber().Length;
+                Assert.Equal(9, snlen);
+
+                byte[] serialNumber = new byte[snlen];
+                Buffer.BlockCopy(cert.GetSerialNumber(), 0, serialNumber, 0, snlen);
+
+                Assert.Equal(0x9e, serialNumber[0]);
+                Assert.Equal(0x7a, serialNumber[1]);
+                Assert.Equal(0x5c, serialNumber[2]);
+                Assert.Equal(0xcc, serialNumber[3]);
```

Comment:
Assert.Equal("9E7A5CCC...", serialNumber.ByteArrayToHex()); is both more compact and (I think) more traceable (in particular, it shows the context of what went wrong, for endianness, or padding byte, or whatever).